homeassistant/components/zha/config_flow.py

```diff
@@ -72,6 +77,41 @@ def _format_backup_choice(
     return f"{dt.as_local(backup.backup_time).strftime('%c')} ({identifier})"
 
 
+async def list_serial_ports(hass: HomeAssistant) -> list[ListPortInfo]:
+    """List all serial ports, including the Yellow radio and the multi-PAN addon."""
+    ports = await hass.async_add_executor_job(serial.tools.list_ports.comports)
+
+    # Add useful info to the Yellow's serial port selection screen
+    try:
+        hardware.async_info(hass)
+    except HomeAssistantError:
+        pass
+    else:
+        yellow_radio = next(p for p in ports if p.device == "/dev/ttyAMA1")
```

Comment:
shouldn't you check that the system is  a yellow too?

Reply:
`hardware.async_info(hass)` is provided by `homeassistant.components.homeassistant_yellow`, which explicitly checks the board name.